### PR TITLE
6.2 r2 release docs

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

**Review by jpetersonssr — APPROVED** [review]

File: docs/release_notes_128t_6.2.md [hunk 1]
```diff
@@ -24,6 +24,39 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-46120 HA Fabric Warning message supression in Azure:** In Azure, it is not possible to configure a non-forwarding fabric interface on the SSR, thus this error will be present on every commit. This message is now supressed.
+------
+- **I95-47041 Selection of Mist Cloud instance during whitebox onboarding:** The onboarding interface now silently queries all Mist Instances and provides a drop down selector to allow login to the appropriate Mist instance (Global01, Global03, EU, etc.).
+------
+- **I95-47253 Stuck Session Detection:** In situations where forward traffic is received, but there is no reverse traffic; for example, when the local IP of an interface performing source-nat changes, or when the local IP of an interface changes while sending traffic over SVR to a waypoint, the SSR will now mark the session for a flow-move with new reverse flow needed. If the criteria is met, the source-NAT or waypoint  will be updated with the correct information on the next forward packet.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:41]: extra space after waypoint and will

File: docs/release_notes_128t_6.2.md [hunk 2]
```diff
@@ -24,6 +24,39 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-51614 Common Criteria Certification - Firewall Protection Profile :** Several updates to existing functionality have been made to address Common Criteria Certification. For additional information, see [Customizable Firewall Rules and Filters](sec_firewall_filtering.md).
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:43]: We can't claim certification yet. Instead, call out each of the firewall features that have been added. Add each one as a new feature, but use the same Jira ID for each. Add a brief description for each feature.

Packet Filtering
ICMP Filtering
IPv4 Option Filtering
TCP Half-Open Connection Limit

File: docs/release_notes_128t_6.2.md [hunk 3]
```diff
@@ -24,6 +24,39 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+- **I95-51648 Application ID Database Memory Improvements:** Internal improvements have been made reduce memory consumption of the Application Identification IP database.
+------ 
+- **I95-53045 Increase default redundancy and capacity settings:** The default values have been adjusted for deployments with large configurations and potentially high latency between nodes. 
+------
+- **I95-53820 MD5 Authentication for MSDP:** An `auth` password option has been added to the [`msdp peer`](config_command_guide.md#configure-authority-router-routing-msdp-peer) and the [`msdp mesh-group`](config_command_guide.md#configure-authority-router-routing-msdp-mesh-group) configurations to support MD5 authentication. For more information, refer to the MSDP command documentation linked above. 
+
+### Resolved Issues
+
+- **The following CVEs have been resolved in this release:**
+
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:54]: - [ ] TODO. Add CVEs here.

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_6.2.md [hunk 4]
```diff
@@ -50,12 +57,72 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-54265 Schema not included in API response for a template:** The `schema` object has been added to the API. 
+------
+- **I95-54294 Unable to delete capture-filter created with `&&` operator:** Resolved an issue that disallowed deleting capture-filters containing `&&`. Customers on older versions of software can work around this by creating capture-filters using `and` instead of `&&`.
+------
+- **I95-54398 ASM - Prune flag set incorrectly:** Resolved an issue with how outgoing interfaces in the mroute entry are reported. If the outgoing interface list only contains PIM SVR interfaces, the P flag was displayed incorrectly. The outgoing interface list is populated correctly, and the P flag now shows correctly.
+------
+- **I95-54434 MistPCAP failure - inverted commands observed by the device:** In rare cases where captures are created and deleted too quickly, a delete command may be received by the device before the create command. These timing issues have been resolved.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:121]: Need space after Mist

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_6.2.md [hunk 5]
```diff
@@ -118,12 +117,16 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+- **I95-54434 Mist PCAP failure - inverted commands observed by the device:** In rare cases where captures are created and deleted too quickly, a delete command may be received by the device before the create command. These timing issues have been resolved.
 ------
 - **I95-54490 Permission denied when trying to open a user config file:** Resolved a permissions issue for the `connect router` command by adding ACLs for reverse SSH nodeIdentifier so that this is accessible for admin users.
 ------
 - **WAN-1958 Mist agent crashes:** Increased internal file system limits which were preventing some services from starting correctly at boot. Limits were raised based on expected system usage.
 
+### Caveats
+
+- **I95-54780 Forwarding Utilization stats are missing:** This issue is the result of the fix in place for the `stats default retention short` setting was not being honored (I95-53875). This will be resolved in the next patch release.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:128]: Specifically passed one hour. Suggested rewording:
Forwarding Utilization stats are not retained beyond one hour:

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_6.2.md [hunk 6]
```diff
@@ -79,12 +79,14 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+- **I95-53666 Unable to create webserver certificate request:** This issue has been resolved by providing ACL permissions for the webserver to the certificate directory.  
 ------
 - **I95-53777 Multicast traffic not passing after an HA failover:** Resolved an issue where the multicast service next hops were not reloading on a configuration change. 
 ------
 - **I95-53787 Stats not present on conductor:** Running show device-interface router all on a conductor caused stats (in-octets, in-unicast-pkts, etc...) to be incorrectly displayed as "n/a" instead of the correct value. This issue has been resolved.
 ------
+- **I95-53851 DHCP server scripts return false positive:** During the creation process, the DHCP scripts check the server name for an existing namespace. The script would return a false positive if the configured name was a substing of an existing name. Resolved an issue where a DHCP server won't come up if the device port is a substring of another device port that is also configured as a DHCP server.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:88]: Typo: substing

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

**Review by MichaelBaj — APPROVED** [review]

File: docs/release_notes_128t_6.2.md [hunk 7]
```diff
@@ -24,6 +24,111 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-53858 Active sessions counter continuously incrementing:** The SSC active sessions counter has been updated to correctly handle session removal.
+------
+- **I95-53875 The `show stats service-area sent success` metric was retained longer than needed:** Resolved an issue where the `stats default retention short` setting was not being honored.
+------
+- **I95-53907 SSR readvertising SA to MSDP mesh peers:** Resolved an issue where MSDP SA's received from a mesh-group peer were being re-advertised to the mesh-group. This issue has been resolved by verifying sender of the SA.
+------
+- **I95-53915 Removing an X710/X722 port from SSR can cause high RX latency:** Deleting an SSR interface from an X710/722 NIC which is part of a multi-port device introduces RX latency into sibling ports until the SSR is restarted. This has been resolved by enabling the i40e “multi-driver” mode to preserve global registers that are shared across ports. 
+------
+- **I95-53896 nodeMonitor failed to get data for show platform disk:** Some of the dynamic access for smartctl objects were not protected. A check for the object existence has been added before attempting to read it.
```
Comment (jeffmart-jnpr) [inline docs/release_notes_128t_6.2.md:100]: The number is inverted, this is https://128technology.atlassian.net/browse/I95-53986

File: docs/release_notes_128t_6.2.md [hunk 8]
```diff
@@ -24,6 +24,111 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+	* **IPv4 Option Filtering:** The SSR has the ability to go deeper than the basic IPv4 header options check and inspect the options to make necessary decisions whether the packets are allowed or dropped and logged.
+	* **Transport State Enforcement:** By default, the SSR checks and follows the TCP sequence numbers of all the sessions passing through, and increments the associated metrics. Setting the Transport State Enforcement field to Strict ensures any packets in the TCP stream that fall outside of the sequence number stream will be dropped.
+	* **TCP Half-open Connection Limit:** Half-open TCP connections are those where the handshake has started but not completed. The SSR provides the ability to configure a limit to these half-open TCP connections.
+------
+- **I95-51648 Application ID Database Memory Improvements:** Internal improvements have been made reduce memory consumption of the Application Identification IP database.
```
Comment (jeffmart-jnpr) [inline docs/release_notes_128t_6.2.md:52]: Recommend rewording this section:
"have been made reduce memory consumption"

to:

"have been made to reduce memory consumption"

File: docs/release_notes_128t_6.2.md (hunk 7 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_6.2.md:100]: This has an exclude on it - I just found it now. Unless you have objections, I will remove it.

File: docs/release_notes_128t_6.2.md [hunk 9]
```diff
@@ -24,6 +24,111 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-38188 Repurposing an HA conductor to a standalone conductor left services for the second conductor:** Resolved an issue where the reverse SSH tunnels from a managed router to the second HA conductor node were not cleaned up if the conductor was converted back to a standalone conductor. The salt states now stop services to a second conductor when it is removed from the HA configuration. 
+------
+- **I95-48783 Conductor process logs are unbounded, risking storage exhaustion:** auditd logs consuming the disk space when the node monitor is in a disconnected state and the audit logs are left unconsumed. There was a limit to the log file size, but not the number of files. The number of files is now limited.
+------
+- **II95-50493 Memory calculation for alarms is confusing:** This alarm was designed to trigger when memory usage went above 90% and clear only when memory usage went below 80%, causing confusion. The memory usage alarm no longer requires memory usage to go below 80% to clear; it will clear when memory usage goes below 90%.
```
Comment (jeffmart-jnpr) [inline docs/release_notes_128t_6.2.md:68]: There is a double I here

**Review by MichaelBaj — APPROVED** [review]

**Review by jeffmart-jnpr — APPROVED** [review]

**Pull request merged**